radar-base.sh

```diff
@@ -28,6 +28,21 @@ get_fetch_time() {
 
 }
 
+get_prompt_prefix_and_suffix_strings() {
+  if [ -f "$rcfile_path/.gitradarrc.bash" ]; then
+    source "$rcfile_path/.gitradarrc.bash"
+  elif [ -f "$rcfile_path/.gitradarrc.zsh" ]; then
+    source "$rcfile_path/.gitradarrc.zsh"
+  elif [ -f "$rcfile_path/.gitradarrc" ]; then
+    source "$rcfile_path/.gitradarrc"
+  fi
+
+  declare -n ret1=$1
+  declare -n ret2=$2
+  ret1="${GIT_RADAR_PROMPT_PREFIX_STRING:-" "}"
+  ret2="${GIT_RADAR_PROMPT_SUFFIX_STRING:-""}"
```

Comment:
Really interesting approach here, I've never seen this done in Bash. 

However we do already allow customising of prompt "pieces", such as master symbol and and prompt format in [`prepare_bash_colors`](https://github.com/hallzy/git-radar/blob/e796c96256a8b69bb76afaf5b2857d647f9b0a92/radar-base.sh#L46) and [`prepare_zsh_colors`](https://github.com/hallzy/git-radar/blob/e796c96256a8b69bb76afaf5b2857d647f9b0a92/radar-base.sh#L84).

I'd prefer using a similar approach to how we do it there, creating say `PROMPT_PREFIX` and `PROMPT_SUFFIX`. Since this function is only called once, and only after we've set up the colors for rendering the prompt, this will also reduce the number of times we source the rcfiles.